model/pom.xml

```diff
@@ -88,15 +88,26 @@
         <dependency>
             <groupId>org.jboss.arquillian.extension</groupId>
             <artifactId>arquillian-warp</artifactId>
+            <version>1.0.0</version>
             <scope>test</scope>
             <type>pom</type>
             <exclusions>
                 <exclusion>
                     <groupId>org.apache.httpcomponents</groupId>
                     <artifactId>httpcore</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>log4j</groupId>
+                    <artifactId>log4j</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-1.2-api</artifactId>
```

Comment:
I see that this bridge allows arquillian to use log4j2, even though it's coded to use log4j1, but are we including log4j2 anywhere in the poms? See [this link, "Option 1: use the Log4j 1.x bridge (log4j-1.2-api)"](https://logging.apache.org/log4j/2.x/manual/migration.html).

Reply:
We don't, but I don't think we need it: I ran mvn dependency:tree and the log4j2 log4j-api is loaded as a transitive dependency. 

I hadn't seen that link when I made the changes but my process was this:

I excluded the log4j artifact causing the vulnerability when loading the arquillian-warp artifact with a dependency upon it. then I ran the tests and they all failed.
then I made these changes and the tests were successful.

Reply:
I've added the log4j-core artifact now